java/test/src/main/java/io/ray/test/FailureTest.java

```diff
@@ -169,5 +172,16 @@ public void testExceptionSerialization() {
     Assert.assertEquals(ex3.getCause().getClass(), UnreconstructableException.class);
     Assert.assertEquals(((UnreconstructableException) ex3.getCause()).objectId, objectId);
   }
+
+  public void testObjectSerializer() {
```

Comment:
This is too much. Basically what you need is adding one line of code in https://github.com/ray-project/ray/blob/9da7bdcc8ed1700279da31509c40c1632ea05add/java/test/src/main/java/io/ray/test/FailureTest.java#L121